vcd_cli/disk.py

```diff
@@ -41,13 +42,16 @@ def disk(ctx):
         vcd disk info disk1 --id 91b3a2e2-fd02-412b-9914-9974d60b2351
             Get details of the disk named 'disk1' that has the supplied id.
 \b
-        vcd disk create disk1 100 --description '100 MB Disk'
-            Create a new 100 MB independent disk named 'disk1' using the default storage profile.
+        vcd disk create disk1 1000 --description '1 KB Disk'
```

Comment:
I think 
1 KB = 1024 B.
1kB = 1000 B

Reply:
so the notation is KiB for 1024 and KB for 1000